roles/configure-nexus/defaults/main.yml

```diff
@@ -7,5 +7,5 @@ nexus_redhat_script_file: ../files/redhat-repos.json
 configure_labs_repos: true
 nexus_labs_script_name: labs-repos
 nexus_labs_script_file: ../files/labs-repos.json
-nexus_server_poll_retries: 60
-nexus_server_poll_delay_in_seconds: 5
\ No newline at end of file
+nexus_server_poll_retries: 600
```

Comment:
600 x 5 seconds = 50 minutes, and because of the check url scripts being wrong this will time out and fail after 50 minutes.